packages/pluggableWidgets/gallery-web/src/Gallery.tsx

```diff
@@ -23,6 +23,18 @@ export function Gallery(props: GalleryContainerProps): ReactElement {
     const [sortState, setSortState] = useState<SortFunction>();
     const { FilterContext } = useFilterContext();
     const { SortContext } = useSortContext();
+    const isInfiniteLoad = props.pagination === "virtualScrolling";
+    const currentPage = isInfiniteLoad
+        ? props.datasource.limit / props.pageSize
+        : props.datasource.offset / props.pageSize;
```

Comment:
Shouldn't this be incremented by one?
So:

| props.datasource.offset        | props.pageSize           | currentPage  |
| ------------- |-------------| -----:|
| 20      | 5 | You already viewed 4 pages of 5 items, so current page is 5? |
| 25      | 5      |   Already viewed 5 pages of 5 items, so current page is 6? |